tsconfig.json

```diff
@@ -1,18 +1,21 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es2017",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es2015", "dom"],                 /* Specify library files to be included in the compilation. */
+    "target": "es2017" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
```

Comment:
We really should just get rid of all those comments in code.
There is documentation for tsconfig.

We should not have all those comments (including commented out values) but rather just the actual config.

Don't you think?

Reply:
I don't mind either way, for now, this is just tackling the formatting.